BandNameGenerator/ViewModel.cs

```diff
@@ -0,0 +1,54 @@
+﻿using BandNameGenerator.Genres;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BandNameGenerator
+{
+    public class ViewModel
+    {
+        public enum Genre
+        {
+            Metal,
+            Country,
+            Folk
+        }
+
+        public ViewModel()
+        {
+        }
+
+        private Genre selectedGenre = Genre.Metal;
+        public Genre SelectedGenre
+        {
+            get
+            {
+                return selectedGenre;
+            }
+            set
+            {
+                selectedGenre = value;
+                BandName = "";
+            }
+        }
+
+        public string BandName { get; set; }
+
+        public void GenerateName()
+        {
+            BandName = "Best band name evah";
+        }
+
+        private IGenre convertToIGenre(Genre genre)
```

Comment:
I'd name this `NewGenre` or `GetNewGenre` so it's clear it constructs a new object.

I wonder if we should rename the enum to GenereID, to avoid confusion where were dealing with genere (enum) and genera (object).

Reply:
Good point, will change.